src/components/DisplayBlockChain.tsx

```diff
@@ -0,0 +1,45 @@
+/* eslint-disable max-len */
```

Comment:
have a clean linter config instead of making exceptions for eachfile
max len is a good lint rule